src/main/java/uk/gov/hmcts/ccd/auditlog/AuditInterceptor.java

```diff
@@ -59,24 +61,33 @@ private AuditContext populateHttpSemantics(AuditContext auditContext,
         AuditContext context = (auditContext != null) ? auditContext : new AuditContext();
         context.setHttpStatus(response.getStatus());
 
-        if (request.getMethod() != null && httpMethodList.contains(request.getMethod().toUpperCase())) {
-            context.setHttpMethod(request.getMethod());
+        String requestMethod = request.getMethod();
+        if (requestMethod != null && httpMethodList.contains(requestMethod.toUpperCase())) {
+            context.setHttpMethod(requestMethod);
         } else {
             context.setHttpMethod(BAD_VALUE_TOKEN);
             LOG.error("Error while validating Http Method with value {} as it did not meet validation criteria:{}",
```

Comment:
Error while validating Http Method with base-64 value {} as it did not meet validation criteria:{}